README.md

```diff
@@ -1,3 +1,45 @@
 # Neptune - PyTorch integration
 
-See [the official docs](https://docs.neptune.ai/integrations-and-supported-tools/model-training/).
+Experiment tracking for PyTorch-trained models.
+
+## What will you get with this integration?
+
+* Log, organize, visualize, and compare ML experiments in a single place
+* Monitor model training live
+* Version and query production-ready models and associated metadata (e.g., datasets)
+* Collaborate with the team and across the organization
+
+## What will be logged to Neptune?
+
+* Training metrics
+* Model checkpoints
+* Model predictions
+* [Other metadata](https://docs.neptune.ai/logging/what_you_can_log)
+
+![image](https://docs.neptune.ai/img/app/integrations/pytorch.png)
```

Comment:
Interesting. I did not think this would work :D Wonder if it'll update dynamically.